tests/e2e-vcluster/vcluster-ks-0/40-assert.yaml

```diff
@@ -0,0 +1,59 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: v-revive-1-node-sc1
+status:
+  replicas: 1
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: v-revive-1-node-sc2
+status:
+  replicas: 1
+
+---
+apiVersion: v1
+kind: Event
+reason: ReviveDBSucceeded
+source:
+  component: verticadb-operator
+involvedObject:
+  apiVersion: vertica.com/v1beta1
+  kind: VerticaDB
+  name: v-revive-1-node
+---
+apiVersion: vertica.com/v1beta1
+kind: VerticaDB
+metadata:
+  name: v-revive-1-node
+spec:
+  local:
+    dataPath: /my-data
+    depotPath: /my-depot
+    requestSize: 100Mi
+status:
+  subclusters:
+    - addedToDBCount: 1
+      name: sc1
+      # re-ip failed now, after re-ip is fixed, we can check upNodeCount
+      # upNodeCount: 1
```

Comment:
Is this because the image we use in the CI is old? I'm in the process of updating it.

Reply:
Yes, the image doesn't have the NMA endpoints change for revive_db.

Reply:
I guess we can add these back now?

Reply:
Yes, these work now.